src/filesystem/impls/filer/lib/content.js

```diff
@@ -140,7 +141,11 @@ define(function (require, exports, module) {
 
         isResizableImage: function(ext) {
             ext = FilerUtils.normalizeExtension(ext);
-            return ext === '.png' || ext === '.jpg' || ext === '.jpeg';
+            return ext === '.png' || ext === '.jpg' || ext === '.jpeg' || ext === '.gif';
+        },
+
+        isAnimated: function(data) {            
```

Comment:
To avoid confusion between the two uses of `isAnimated`, can you update this to:

```js
isAnimatedImage: function(data) {
    return isAnimated(data);
},
```

Reply:
Ok!